csharp-basic-topics/GuidClassInCSharp/GuidClassInCSharpTests/GuidClassInCSharpUnitTests.cs

```diff
@@ -0,0 +1,54 @@
+using static GuidClassInCSharp.GuidClassInCSharpMethods;
+
+namespace GuidClassInCSharpTests
+{
+    public class Tests
+    {
+        [Test]
+        public void GivenCheckGuidEqualityMethod_WhenEqualGuids_ThenTrue()
+        {
+            var guidA = new Guid("B5724FB4-2FA0-4E68-B108-93C24A1EA6DF");
+            var guidB = new Guid("B5724FB4-2FA0-4E68-B108-93C24A1EA6DF");
+            var checkEquality = CheckGuidEquality(guidA, guidB);
+
+            Assert.That(checkEquality, Is.EqualTo(true));
```

Comment:
Since you are using the constraint model approach of NUnit, this would be better as: Assert.That(checkEquality, Is.True)